.github/workflows/ci.yml

```diff
@@ -9,17 +9,28 @@ on:
       - master
 
 jobs:
-  test-ubuntu:
+  test-debian:
     strategy:
       matrix:
-        ubuntu:
-          - 16.04
-          - latest
-    runs-on: ubuntu-${{ matrix.ubuntu }}
+        image:
+          - ubuntu:latest
+          - ubuntu:16.04
+          - debian:testing
+          - debian:stable
+          - debian:oldstable
+    runs-on: ubuntu-latest
+    container:
+      image: docker.io/library/${{ matrix.image }}
     steps:
+      - name: install dependencies
+        run: |
+          apt-get update &&
+          DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
```

Comment:
Maybe we want to set ENV via the [function for the runner](https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#about-environment-variables). Not sure.

Reply:
I guess it doesn't hurt to set it everywhere, but it's only needed for this particular command so I'd like to keep in there.